urbanopt/geojson-modelica-translator
# Implement a PyPI release workflow

#### Any background context you want to provide?
Releasing to PyPI has several steps that can lead to human error. What if we automated it entirely?
#### What does this PR accomplish?
Implements a GitHub Action to automatically publish to pypi whenever we make a release on Github.
#### How should this be manually tested?
- Make an alpha/beta/rc release and see if it gets up to pypi
- Ensure the repo branch protections are set properly
- Ensure the readme filetype is defined properly or else pypi/twine whines about it
#### What are the relevant tickets?
#541 
#### Screenshots (if appropriate)
